pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = [
+    "setuptools",
+    "wheel",
+]
+
+[tool.black]
+line-length = 79
```

Comment:
why not 80?

Reply:
Nevermind. Strange I thought pep8 would also be 80 chars

Reply:
> The limits are chosen to avoid wrapping in editors with the window width set to 80
https://www.python.org/dev/peps/pep-0008/#maximum-line-length
Also read somewhere that it is for files where the newline character is not counted